Makefile

```diff
@@ -454,6 +454,13 @@ update-dependencies: ## Uses go get -u to update all the dependencies while hold
 	# Copy everything to vendor directory
 	$(GO) mod vendor
 
+vet: ## Run mattermost go vet specifi checks
+	$(GO) get -u github.com/mattermost/mattermost-govet
+	$(GO) vet -vettool=$(GOPATH)/bin/mattermost-govet -license $(TE_PACKAGES)
+
+vet-enterprise: ## Run mattermost go vet specifi checks
```

Comment:
I would like to see just a single target. What do you think about adopting the solution at https://github.com/mattermost/mattermost-server/blob/master/Makefile#L163-L165?

Reply:
I like this proposal, I going to implement it.